src/infra/docs/docs-rs/README.md

```diff
@@ -0,0 +1,14 @@
+# docs.rs
+
+* Source code: [rust-lang/docs.rs][repo]
+* Hosted on: `docsrs.infra.rust-lang.org` (behind the bastion -- [how to connect][bastion-connect])
+* Maintainers: [Joshua Nelson], [Pietro Albini]
```

Comment:
Thinking on this more I think we should just link to the team instead, so that we don't have to update it again when membership changes.
```suggestion
* Maintainers: [docs.rs team]
```